db/migrate/20220810113323_create_posts.rb

```diff
@@ -1,11 +1,11 @@
 class CreatePosts < ActiveRecord::Migration[7.0]
   def change
     create_table :posts do |t|
-      t.integer :Author_id, foreign_key: true
+      t.integer :Author_id
       t.string :Title
       t.text :Text
-      t.integer :CommentsCounter
-      t.integer :LikesCounter
+      t.integer :CommentsCounter, default: 0
+      t.integer :LikesCounter, default: 0
```

Comment:
## [OPTIONAl]

- [ ] Kindly note that the name in convention is extremely important in Ruby on Rails, Only one name in convention error can mess up your code. I would kindly suggest you lowercase and separate words with `_` all of your columns in your tables. For example use `comments_counter` instead of `CommentsCounter`, `Author_id` instead of `author_id`.......